assets/css/admin-notices.scss

```diff
@@ -31,6 +31,11 @@ $sensei-notice-icon-width: 30px;
 		}
 	}
 
+	&__content {
+		color: #101517;
+		max-width: 99%;
```

Comment:
This is opinionated. Without this, notice text seems to run up too closely with the action buttons, and I thought more whitespace looked nicer. However, if we think differently, this need not stay as is.

Reply:
Maybe we could use a `padding-right` instead?